packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -73,26 +83,57 @@ class ScrollbarPainter extends ChangeNotifier implements CustomPainter {
   /// Mustn't be null.
   final Animation<double> fadeoutOpacityAnimation;
 
-  /// Distance from the scrollbar's start and end to the edge of the viewport in
-  /// pixels. Mustn't be null.
+  /// Distance from the scrollbar's start and end to the edge of the viewport
+  /// in pixels. It affects the quantity of available scrolling area.
+  ///
+  /// Mustn't be null and defaults to 0.
   final double mainAxisMargin;
 
-  /// Distance from the scrollbar's side to the nearest edge in pixels. Must not
-  /// be null.
+  /// Distance from the scrollbar's side to the nearest edge in pixels.
+  ///
+  /// Must not be null and defaults to 0.
   final double crossAxisMargin;
 
   /// [Radius] of corners if the scrollbar should have rounded corners.
   ///
   /// Scrollbar will be rectangular if [radius] is null.
   final Radius radius;
 
-  /// The smallest size the scrollbar can shrink to when the total scrollable
-  /// extent is large and the current visible viewport is small, and the
-  /// viewport is not overscrolled. Mustn't be null.
+  /// The amount of space by which to inset the scrollbar's start and end, as
+  /// well as its side to the nearest edge, in pixels.
+  ///
+  /// This is typically set to the current [MediaQueryData.padding] to avoid
+  /// partial obstructions such as display notches. Contents that overlap
```

Comment:
Hmm I guess I was trying to explain how `padding` is different from `mainAxisMargin`. Contents that are covered by the `padding` area is considered not visible when calculating the content's visible percentage while `mainAxisMargin` doesn't affect the visible percentage. Does "[padding] reduces the visible content size to total content size ratio during the scrollbar size calculation, thus reduces the size of the scrollbar. If you only want additional margins around the scrollbar, see [mainAxisMargin], which does not affect the said ratio." sound better, or should I take the second sentence off since it's not too helpful?

Reply:
Removed the second sentence.